app/controllers/users.js

```diff
@@ -0,0 +1,18 @@
+const logger = require('../logger');
+
+const { hashPassword } = require('../helpers/encrypt');
+const { createUser } = require('../services/users');
+
+exports.signUp = (req, res, next) => {
```

Comment:
This is good, but async / await better :)

Reply:
I agree, it was already changed